api/app/Models/User.php

```diff
@@ -220,14 +221,25 @@ public function filterByPools(Builder $query, array $poolCandidates): Builder
                     }
                   })
                   ->where(function ($query) use ($poolCandidates) {
-                    if (array_key_exists('statuses', $poolCandidates) && !empty($poolCandidates['statuses'])) {
+                    if (!empty($poolCandidates['statuses'])) {
                         $query->whereIn('pool_candidates.pool_candidate_status', $poolCandidates['statuses']);
                     }
                   });
         });
 
         return $query;
     }
+    public function filterApplicantByPools(Builder $query, array $pools): Builder
+    {
+        $poolCandidates = [
```

Comment:
We do currently use filtering by default for Pool Candidates (that's what the scopes field in the poolCandidates and searchPoolCandidates is for) but they could be changed over to be defaulted in the GraphQL as well. I decided not to add those fields to this query because I thought we might like to prevent people being able to get a count of all expired or 'no longer interested' candidates but if we don't care about that I can do that instead.

Reply:
Ok, I won't worry about it then. I'm not sure we want to use `UserPoolFilterInput` though since it's default status is empty/all and the default here should be available.

Reply:
I think creating a new input with a different default is a bit much... but I'm satisfied since the default is explicit in the schema and it is possible to override it to query all applicants. :laughing: Let's go with it.